demo/src/index.js

```diff
@@ -49,33 +41,32 @@ class App extends Component {
         hidden: true,
       },
     ],
-    baseDomain,
+    baseDomain: _baseDomain,
+    zoomable: true,
   };
 
   componentDidMount() {
     setTimeout(() => {
-      console.log('Updated state');
       this.setState({
         baseDomain: this.state.baseDomain.map(p => p + 1000 * 60 * 15),
+        zoomable: false,
       });
     }, 10000);
   }
 
-  onMouseMove = points => {};
-
   render() {
-    const { series } = this.state;
+    const { series, baseDomain, zoomable } = this.state;
     return (
       <DataProvider
         defaultLoader={loader}
         series={series}
         yAccessor={d => d.value}
         xAccessor={d => d.timestamp}
         yAxisWidth={50}
-        baseDomain={this.state.baseDomain}
+        baseDomain={baseDomain}
       >
         <div style={{ height: '800px' }}>
```

Comment:
Do we still need `height: 800px` here?

Reply:
Thanks for quick feedback. Yes -- `<LineChart>` will go full width but currently a set height. New PR could be dynamic height.